frontend/public/locales/fr/apply.json

```diff
@@ -47,38 +47,39 @@
-        "considered-minor": "(FR) If you are considered a minor in your province or territory, your information can also be shared with your parents or legal guardians.",
-        "analysis": "(FR) ESDC may use or share the information you provide for policy analysis, research, monitoring and/or evaluation activities within ESDC, Health Canada and Statistics Canada. These uses or activities will not result in any administrative decisions made about you."
+        "heading": "Partage d'informations et prestataires de santé bucco-dentaire",
+        "enrol-consent": "En vous inscrivant au RCSD, vous consentez également à ce que vos fournisseurs de soins de santé buccodentaire divulguent tous les renseignements personnels, tel qu'autorisé par la <i>Loi sur le ministère de la Santé</i> et la <i>Loi sur les mesures de soins dentaires</i>, qui peuvent être jugés utiles par la Sun Life dans le contexte de l'administration du programme, de la vérification et de la gestion des demandes de règlement, et de la sécurité des patients/clients. Ces renseignements peuvent comprendre, sans s'y limiter, des détails sur les services fournis, des données cliniques sur le patient, des paiements et des radiographies.",
+        "considered-minor": "Si vous êtes considéré comme mineur dans votre province ou territoire, vos renseignements peuvent également être communiqués à vos parents ou tuteurs légaux.",
+        "analysis": "EDSC peut utiliser ou communiquer les renseignements que vous fournissez pour les activités d'analyse des politiques, de recherche, de surveillance et d'évaluation au sein d'EDSC, de SC et de Statistique Canada. Ces utilisations ou activités n'auront aucune incidence sur les décisions administratives prises à votre sujet."
       }
     },
     "apply": {
-      "application-participation": "(FR) Participation in the Canadian Dental Care Plan is voluntary. However, if you do not provide your personal information in your application, you cannot be considered for the Canadian Dental Care Plan.",
-      "application-consent": "En poursuivant la demande, vous acceptez aux conditions générales d'utilisation ci-dessus et à l'énoncé de confidentialité. ",
+      "application-participation": "La participation au Régime canadien de soins dentaires est volontaire. Toutefois, si vous ne fournissez pas vos renseignements personnels dans votre demande, votre admissibilité au Régime canadien de soins dentaires ne pourra pas être validée.",
+      "application-consent": "En poursuivant cette demande, vous reconnaissez avoir lu et accepté les modalités et l'énoncé de confidentialité cidessus. De plus, vous consentez à ce que vos renseignements personnels soient communiqués aux fins susmentionnées.",
```

Comment:
`cidessus` should be `ci-dessus`

Reply:
I added a [comment](https://www.figma.com/design/pXiZz8YkzcMY5S5aq0B4ce?node-id=12172-58867#840741893) on Figma.

Reply:
Added hyphen